src/Cake.Common/Tools/NUnit/NUnit3AppDomainUsage.cs

```diff
@@ -0,0 +1,31 @@
+namespace Cake.Common.Tools.NUnit
+{
+    /// <summary>
+    /// The /domain option controls of the creation of AppDomains for running tests.
+    /// </summary>
+    public enum NUnit3AppDomainUsage
+    {
+        /// <summary>
+        /// Create a separate AppDomain for each assembly if more than one is listed on the command
+        /// line, otherwise creates a single AppDomain
+        /// </summary>
+        Default,
```

Comment:
Would prefer this enum value to explicitly be assigned the value of `0` to avoid problems if someone reorder this in the future.